frontend/src/store/gym/action.ts

```diff
@@ -35,7 +35,7 @@ export const getGymHistory = () => async (dispatch: Dispatch<ActionTypes>) => {
 }
 
 export const moveKey = (userID: string, name: string, telegram: string) => async (dispatch: Dispatch<ActionTypes>) => {
-  await post(ENDPOINTS.MOVE_KEY, DOMAINS.GYM, {}, {}, `/${userID}`)
+  await post(ENDPOINTS.MOVE_KEY, DOMAINS.GYM, {}, {}, `/${userID}?token=${localStorage.getItem('token')}`)
```

Comment:
https://washpost.engineering/string-manipulation-of-urls-is-an-anti-pattern-e2b341129025

Consider reading this (and related articles) and sharing w ur peers.

Reply:
Discussed this with Richard, I think this one for now we just keep it, because we would need to change the `makeRequest` in `endpoints.ts` which might be a little dangerous since all our API calls rely on it. Will probably put this is a seperate PR since we do this in other numerous functions as well?